Quoridor.Model/AlivePlayer.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace Quoridor.Model
+{
+    internal class Player : IPlayer
+    {
+        public Player(Wall[] walls, Position start, Position end, Position currentPlace)
+        {
+            currentCell = currentPlace;
+            endPosition = end;
+            Walls = walls;
+            startPosition = start;
+        }
+        public Wall[] Walls { get; private set; }
+        public Position currentCell { get; private set; }    
+        public Position startPosition { get; private set; }
+        public Position endPosition { get; private set; }
+        public bool HasWall { get; private set; }
```

Comment:
Not **HasWall**, but **HasWon**

Reply:
There is HasWon method below, I added this HasWall field for PlaceWall method. Should I remove it?

Reply:
I think so, because just [above ](https://github.com/Sierra-KPI/Quoridor/pull/10#discussion_r716197857)I suggested another way of counting walls and @Anastasia-Tre agreed with it

Reply:
But You can wait for tomorrow and then We will discuss it more

Reply:
It's not connected with counting walls, it's just a way to check if that cell already has wall in it.

Reply:
Oh, okay, then it's My bad.
Sorry

Reply:
No problem) I'm glad that you check code so thoroughly.